script.js

```diff
@@ -0,0 +1,93 @@
+const myLibrary = [];
+const table = document.getElementById('table');
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+Book.prototype.changeRead = function () {
+  this.read = !this.read;
+};
+
+function addBookToLibrary(book) {
+  myLibrary.push(book);
+  book.id = myLibrary.indexOf(book);
+}
+
+function deleteBook(book) {
+  myLibrary.splice(book.id, 1);
+  const row = document.getElementById(`bookRow${book.id}`);
+  row.remove();
+  return myLibrary;
+}
+
+function updateStatusButton(button, book) {
+  button.innerHTML = book.read ? 'Read' : 'Not read yet';
+}
+
+function bookStatusButton(book) {
+  const bookStatusButton = document.createElement('button');
+  const readCell = document.getElementById(`readCell${book.id}`);
+  bookStatusButton.innerHTML = book.read ? 'Read' : 'Not read yet';
+  readCell.appendChild(bookStatusButton);
+  bookStatusButton.addEventListener('click', () => {
+    book.changeRead();
+    updateStatusButton(bookStatusButton, book);
+  });
+}
+
+function addDeleteBtn(book) {
+  const deleteBtn = document.createElement('button');
+  const deleteCell = document.getElementById(`deleteCell${book.id}`);
+  deleteBtn.innerHTML = 'Delete';
+  deleteCell.appendChild(deleteBtn);
+  deleteBtn.addEventListener('click', () => {
+    deleteBook(book);
+  });
+}
+
+function addRow(book) {
+  const row = table.insertRow();
+  row.id = `bookRow${book.id}`;
+  const titleCell = row.insertCell();
+  const authorCell = row.insertCell();
+  const pagesCell = row.insertCell();
+  const readCell = row.insertCell();
+  const deleteCell = row.insertCell();
+  deleteCell.id = `deleteCell${book.id}`;
+  readCell.id = `readCell${book.id}`;
+  titleCell.innerHTML = book.title;
+  authorCell.innerHTML = book.author;
+  pagesCell.innerHTML = book.pages;
+  bookStatusButton(book);
+  addDeleteBtn(book);
+}
```

Comment:
This a very impressive way of displaying your items 👏🏿 . I absolutely love it. However, you need to use a loop to display the items in your array. Kindly check below ⬇️ 

- [ ] Kindly use a loop to display the items in your array. This is a hard requirement for this project as it will help you practice loops In javascript. ⬇️ 

![books_render_loop](https://user-images.githubusercontent.com/32361963/115109887-237e2800-9f81-11eb-8494-b19c8b3e84ff.png)